packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -369,9 +369,9 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
     if (this.props.componentUrl) {
       mdUrl = `${this.props.componentUrl}/docs/${componentName}${section}.md`;
       // Replace /tree/ or /blob/ with /edit/ to get straight to GitHub editor.
```

Comment:
muahaha! No change file is failing the build! Yay!

Reply:
Heh.. forgot to push :(